src/extension/content-script/batteries/helpers.ts

```diff
@@ -7,7 +7,9 @@ import { ExtensionIcon } from "../../background-script/actions/setup/setIcon";
 export const findLightningAddressInText = (text: string): string | null => {
   // The second lightning emoji is succeeded by an invisible
   // variation selector-16 character: https://regex101.com/r/Bf2GpN/1
```

Comment:
Is the regex link still matching this? Should it?

Reply:
Yeah thought of it before, it just has the regex of two lightning emojis so that we won't have to change